src/components/material/material-buttons/online/MaterialButtonOnlineExternal.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import { FC } from "react";
+import { useText } from "../../../../core/utils/text";
+import { LinkNoStyle } from "../../../atoms/link-no-style";
+import { Button } from "../../../Buttons/Button";
+
+export interface MaterialButtonOnlineExternalProps {
+  externalUrl: string;
+  origin: string;
+}
+
+const MaterialButtonOnlineExternal: FC<MaterialButtonOnlineExternalProps> = ({
+  externalUrl = "https://google.com",
+  origin
+}) => {
+  const t = useText();
+  const externalLinkObject = new URL(externalUrl);
+
+  return (
+    <LinkNoStyle url={externalLinkObject}>
+      <Button
```

Comment:
I (and Jesper) am not a fan of concatenating strings, because this can render them untranslatable. I'll approve, but perhaps make a todo, and we'll talk about this at the next architecture meeting

Reply:
Great point! I added it to the agenda for the next architecture meeting :) 
Let's talk about it.